src/main.py

```diff
@@ -26,15 +26,9 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(LOGGING_LEVEL)
 
-# We need an authorized token to fetch the API. If you have SSO on your org you will need to grant permission
-# Your app and the Github webhook will share this secret
-# You can create an .env file and place the token in it
-GH_TOKEN = os.environ.get("GH_TOKEN")
-# Where to report Github actions transactions
-SENTRY_GITHUB_DSN = os.environ.get("SENTRY_GITHUB_DSN")
+handler = WebAppHandler()
```

Comment:
It seems we missed this change in the previous PR.

I wish I had a proper test to see the web app simply start without failing. Let me see what I can do.